src/audio/pipewire/SDL_pipewire.c

```diff
@@ -93,6 +93,13 @@ static int (*PIPEWIRE_pw_properties_setf)(struct pw_properties *, const char *,
 
 #ifdef SDL_AUDIO_DRIVER_PIPEWIRE_DYNAMIC
 
+SDL_ELF_NOTE_DLOPEN(
+    "audio-libpipewire",
+    "Support for audio through libpipewire",
+    SDL_ELF_NOTE_DLOPEN_PRIORITY_RECOMMENDED,
```

Comment:
I also bumped this to recommended.

Reply:
Do we really want pipewire to be recommended and pulseaudio only suggested? I would have assumed the other way around, if there's any difference between the two - libpulse is pretty much the de facto standard audio API on Linux (and Pipewire implements PulseAudio's protocol well).

Reply:
Oh, I assumed because pipewire emulated pulseaudio, that pulseaudio was on the way out.
I'll set both to recommended then.